poseidon/controllers/faucet/acls.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 4 March 2020
+@author: Charlie Lewis
+"""
+import logging
+
+import yaml
+
+from poseidon.controllers.faucet.helpers import yaml_in
+from poseidon.controllers.faucet.helpers import yaml_out
+from poseidon.helpers.exception_decor import exception
+
+
+class ACLs:
+
+    def __init__(self):
+        self.logger = logging.getLogger('acls')
+
+    def apply_acls(self, config_file, rules_file, endpoints, force_apply_rules,
+                   force_remove_rules, coprocess_rules_files, obj_doc,
+                   rules_doc):
+        if not endpoints:
+            return obj_doc
+
+        # get acls file and add to faucet.yaml if not already there
+        if 'include' not in rules_doc:
```

Comment:
apply_acls() is pretty big - perhaps this section here that brings in include rules could be in its own function?